Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.yml

```diff
@@ -2394,6 +2394,2519 @@ script:
+      isArray: false
+      name: certificate_file
+      required: false
+    - description: Multi certificate (dependencies use fortiwebvm-multi-certificate-list to get all the multi-certificates). Relevant when enable_ssl=enable and certificate_file=Multi Certificate. Supports API version 2 only.
+      isArray: false
+      name: multi_certificate
+      required: false
+    - description: Relevant when enable_ssl=enable and certificate_file=Letsencrypt (dependencies - use fortiwebvm-letsencrypt-certificate-list command to list all the letsencrypt certificates). Supports API version 2 only.
+      isArray: false
+      name: letsencrypt
+      required: false
+    - description: "The name of a group of intermediate certificate authority (CA) certificates, if any, that FortiWeb presents to clients to complete the signing chain for them and validate the server certificate’s CA signature (dependencies - use fortiwebvm-certificate-intermediate-group-list command to get all the certificate intermediate groups). Relevant when ssl=enable."
+      isArray: false
+      name: certficate_intermediate_group
+      required: false
+    - description: 'The client certificate that FortiWeb uses to connect to this server pool member (dependencies - use fortiwebvm-local-certificate-list command to get all the local certificates). Relevant when: ssl=enable.'
+      isArray: false
+      name: client_certificate_file
+      required: false
+    - auto: PREDEFINED
+      description: Enable to use the health check specified by health in the server pool configuration. Disable to use the health check specified by health in this pool member configuration. Supports API version 2 only.
+      isArray: false
+      name: health_check_inherit
+      predefined:
+      - enable
+      - disable
+      required: false
+    - description: 'The domain name of the server pool. Required when health_check_inherit=enable. Supports API version 2 only.'
+      isArray: false
+      name: health_check_domain
+      required: false
+    - description: The name of a server health check FortiWeb uses to determine the responsiveness of server pool members (dependencies - use fortiwebvm-server-health-check-list command to get all health check policies). Supports API version 2 only.
+      isArray: false
+      name: health_check
+      required: false
+    deprecated: false
+    description: "Create a rule for an true transparent server pool group (server pool rule is a member of server pool group). True transparent proxy  -  Requests are destined for a real web server instead of the FortiWeb appliance. Supports API versions 1 & 2. Server pool rule advanced SSL settings dose not supported in this command."
```

Comment:
```suggestion
    description: "Create a rule for a true transparent server pool group (server pool rule is a member of server pool group). True transparent proxy  -  Requests are destined for a real web server instead of the FortiWeb appliance. Supports API versions 1 & 2. Server pool rule advanced SSL settings are not supported in this command."
```